plugins/rikki/loungeviews/components/TimelineEntries.php

```diff
@@ -107,7 +107,8 @@ public function onRender()
                     break;
                 case 'division':
                     $timeline = new Collection([]);
-                    Timeline::with('divisions', 'matches.teams.divisions', 'teams.divisions', 'sloths.teams', 'teams.logo')->orderBy('created_at', 'desc')->chunk(100, function ($someTimelines) use ($id, $timeline) {
+                    Timeline::with('divisions', 'matches.teams.divisions', 'teams.divisions', 'sloths.teams.divisions', 'teams.logo')->orderBy('created_at', 'desc')
+                    ->chunk(100, function ($someTimelines) use ($id, $timeline) {
```

Comment:
When you are splitting a line, the new lines should start 2*4=8 spaces further inwards than the original line. (Also, the first line is still super-long, 80 characters is generally a good guideline)